package.json

```diff
@@ -161,6 +168,8 @@
     "serve-index": "^1.9.1",
     "stats-lite": "^2.2.0",
     "terser-webpack-plugin": "^3.1.0",
+    "ts-node": "^8.3.0",
+    "tslib": "^1.10.0",
```

Comment:
Is `tslib` needed here since it's also added to the angular package.json?

Reply:
We don't need this for the library. but needed anyway for E2E app. lets keep it as it is since we eventually have to add it back.